recipes-core/images/intel-aero-image.bb

```diff
@@ -61,6 +61,18 @@ GRUB_TIMEOUT = "3"
 IMAGE_INSTALL += "librealsense"
 IMAGE_INSTALL += "librealsense-graphical-examples"
 
+#connectivity
+IMAGE_INSTALL += "backport-iwlwifi"
+IMAGE_INSTALL += "linux-firmware-iwlwifi-8000c"
+IMAGE_INSTALL += "hostapd"
+IMAGE_INSTALL += "autostart-hostapd"
+IMAGE_INSTALL += "autostart-supplicant"
+
+# LTE MODEM
+IMAGE_INSTALL += "glibc-gconvs glibc-utils glibc-gconv-iso8859-1 modemmanager \
+	opkg opkg-utils rpm icon-naming-utils libtool libndp libnl libinput \
```

Comment:
opkg opkg-utils rpm? the opkg packages must be wrong.  And these libraries should be dependency from the tools, no need to add them here